static/src/actions/auth.js

```diff
@@ -120,10 +120,10 @@ export function registerUserFailure(error) {
     };
 }
 
-export function registerUser(email, password) {
+export function registerUser(profile, email, password) {
     return function (dispatch) {
         dispatch(registerUserRequest());
-        return create_user(email, password)
+        return create_user(profile, email, password)
```

Comment:
You'll also need to update the create_user function which is inside utils/http_functions.js. Not sure why it's where, from the original boilerplate...

Reply:
Ooops no worries you did this